docs/network-policy.md

```diff
@@ -0,0 +1,361 @@
+
+  Each Namespace creates an AddressSet to which the IPs of all pods in that Namespace are added. This is used in NetworkPolicy `ingress` and `egress` sections to implement the namespace selector.
+
+  Each element in the `from` or `to` list results in an AddressSet containing the IP addresses of all peer pods(i.e all pods touched by this policy) As pods are created, updated, or deleted each AddressSet is updated to add the new pod if it matches the selectors, or to remove the pod if it used to match selectors but no longer does. Namespace label changes may also result in AddressSet updates to add or remove pods if the namespace now matches or no longer matches the `namespaceSelector`.
+  
+  If an `ipBlock` is specified, an ACL with the label `ipblock_cidr="false"` is added to the policy's PortGroup with `priority=1001` that allows traffic to or from the list of CIDRs in the `ipBlock`, any exceptions are added as `drop` ACLs to the policy's PortGroup with `priority=1010`.
+
+  **Examples:** 
+
+  Given two pods in namespace `default` called  `client1` and `client2` , and one pod in namespace `demo`, called `server` lets make a network policy that allows ingress traffic to the server from `client1` but bocks traffic from `client2` 
+
+  Notice the pod `client1` and namespace `default` are labeled with `app=demo`
+
+  ```
+  [astoycos@localhost demo]$ kubectl get pods -o wide --show-labels --all-namespaces
+  NAMESPACE            NAME                                        READY   STATUS    RESTARTS   AGE     IP           NODE                NOMINATED NODE   READINESS GATES   LABELS
+  default              client1                                     1/1     Running   0          5m5s    10.244.2.5   ovn-worker          <none>           <none>            app=demo
+  default              client2                                     1/1     Running   0          4m59s   10.244.1.4   ovn-worker2         <none>           <none>            <none>
+  demo                 server                                      1/1     Running   0          42s     10.244.2.6   ovn-worker          <none>           <none>            <none>
+  ```
+
+  ```
+  [astoycos@localhost demo]$ kubectl get namespace --show-labels
+  NAME                 STATUS   AGE   LABELS
+  default              Active   94m   ns=default
+  demo                 Active   66m   <none>
+  ```
+
+  Before applying the following network policy both pods `client1` and `client2` can reach the `server` pod
+
+  ``` 
+  apiVersion: networking.k8s.io/v1
+  kind: NetworkPolicy
+  metadata:
+    name: allow-from-client
+  spec:
+    podSelector: {}
+    policyTypes:
+    - Ingress
+    ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            ns: default
+        podSelector:
+          matchLabels:
+            app: demo
+  ```
+
+  after applying this Network policy in namespace `demo` only the pod `client1` can reach the `server` pod 
```

Comment:
I think it would be useful to also add the command here, something like `kubectl create -f policy.yaml`